server/app/views/admin/TranslationFormView.java

```diff
@@ -83,20 +84,33 @@ protected final FormTag renderTranslationForm(
       Http.Request request,
       Locale locale,
       String formAction,
-      ImmutableList<DomContent> formFieldContent) {
+      ImmutableList<DomContent> formFieldContent,
+      boolean isProgramEdit) {
+    String redirectUrl =
+        isProgramEdit
+            ? controllers.admin.routes.AdminProgramController.index().url()
+            : controllers.admin.routes.AdminQuestionController.index().url();
     FormTag form =
         form()
             .withMethod("POST")
             .with(makeCsrfTokenInputTag(request))
             .withAction(formAction)
             .with(formFieldContent)
             .with(
-                submitButton(
-                        String.format(
-                            "Save %s updates",
-                            locale.getDisplayLanguage(LocalizedStrings.DEFAULT_LOCALE)))
-                    .withId("update-localizations-button")
-                    .withClasses(ButtonStyles.SOLID_BLUE));
+                div()
+                    .withClasses("flex", "flex-row", "gap-x-2")
+                    .with(
+                        a().withHref(redirectUrl)
```

Comment:
Putting a button inside an a tag isn't valid html5 and could cause some weirdness with them both trying to do actions. Simple fix though just move the button styles onto the a tag and remove the button tag.

Reply:
done!